lib/cc/engine/analyzers/javascript/minification_checker.rb

```diff
@@ -0,0 +1,26 @@
+module CC
+  module Engine
+    module Analyzers
+      module Javascript
+        class MinificationChecker
+          MINIFIED_AVG_LINE_LENGTH_CUTOFF = 100
+
+          def initialize(path)
+            @content = File.read(path)
+          end
+
+          def minified?
+            line_count = content.lines.count
+            chars_seen = content.chars.count
```

Comment:
How about just

```rb
ratio = content.chars.count / content.lines.count
ratio >= MINIFIED_AVG_LINE_LENGTH_CUTOFF
```

(Note: I rescind my "white space before return value" suggestion in the case of a 2-line method).